src/services/bindService.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+angular.module("openshiftCommonServices")
+  .service("BindingService", function($filter, $q, DataService, DNS1123_SUBDOMAIN_VALIDATION){
+    var bindingResource = {
+      group: 'servicecatalog.k8s.io',
+      resource: 'bindings'
+    };
+
+    var getProjectContext = function(project) {
+      return {
+        namespace: project.metadata.name
+      };
+    };
+
+    var makeBinding = function (serviceToBind) {
+      var generateName = $filter('generateName');
+
+      return {
+        kind: 'Binding',
+        apiVersion: 'servicecatalog.k8s.io/v1alpha1',
+        metadata: {
+          generateName: serviceToBind + '-'
+        },
+        spec: {
+          instanceRef: {
+            name: serviceToBind
+          },
+          secretName: generateName(_.trunc(serviceToBind, DNS1123_SUBDOMAIN_VALIDATION.maxlength - 6) + '-')
+        }
+      };
+    };
+
+    return {
+      bindService: function(project, serviceInstance, appToBind) {
+        var deferred = $q.defer();
+        var context = getProjectContext(project);
```

Comment:
Suggest you take in either `context` or `projectName` as a parameter instead of `project` so that I can call this method needing a project object.